packages/@lightningjs/ui-components/src/components/TextBox/TextBox.d.ts

```diff
@@ -18,27 +18,92 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { StylePartial } from '../../types/lui';
-import type { TextContent } from '../InlineContent/InlineContent';
+import { StylePartial } from '../../types/lui';
+import TextContent from '../InlineContent';
```

Comment:
I don't think InlineContent exports this?

Reply:
I updated TextContent to match the current pattern we have for other components

Reply:
can we also move the TextContent export to the bottom of the file with the other exports? I'd prefer to group all of our exports together in one place